### PR TITLE
Add blog post about lesson infrastructure meeting in November

### PR DESCRIPTION
**Pull request by rgaiacs**
With more than two weeks delay. :-)

### CODE REVIEW

File: _posts/2017/12/2017-12-04-lesson-infrastructure-subcommitee.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+## Non-English Lessons
+
+If you are involved with us since 2014,
+you might remember [this post](https://software-carpentry.org/blog/2014/06/translating-software-carpentry-into-spanish.html) about the attempt to translate the lesson to Spanish
+and [this other post](https://software-carpentry.org/blog/2014/11/korean-translation.html) announcing the lessons in Korean.
+During the meeting,
+we have a conversation about the workflow to translate the lesson to other languages.
+
+Some of the conversation was archived as issues [here](https://github.com/Carpentries-ES/board/issues).
+If you want to get involved with the translation FIXME.
```
Comment (rgaiacs) [inline _posts/2017/12/2017-12-04-lesson-infrastructure-subcommitee.md]: @ErinBecker could you replace the "FIXME" at this line with the correct contact detail before merge this pull request?

Comment (weaverbel) [reply _posts/2017/12/2017-12-04-lesson-infrastructure-subcommitee.md]: @ErinBecker is on leave @rgaiacs so there might be a slight delay with her response.

Comment (rgaiacs) [reply _posts/2017/12/2017-12-04-lesson-infrastructure-subcommitee.md]: @weaverbel Thanks for the information.

@map0logo, @agbeltran, @orchid00 can you provide the preferable contact detail?

**Comment by tracykteal** [comment]
I added in the Spanish lesson information.

**Comment by rgaiacs** [comment]
@tracykteal, Thanks for the information. Looks good to me. Feel free to merge. And could you cross-post it to Data Carpentry website?

**Comment by ErinBecker** [comment]
Sorry @rgaiacs, I was out of work last week due to childcare. Thanks for putting this together.

**Comment by weaverbel** [comment]
It has been posted to both blogs @ErinBecker @rgaiacs

**Pull request merged**